test/units/test_context.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Toshio Kuratomi <tkuratomi@ansible.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division)
+__metaclass__ = type
+
+
+import pytest
+
+from ansible import context
+
+
+class FakeOptions:
+    pass
+
+
+def test_set_global_context():
+    options = FakeOptions()
+    options.tags = [u'production', u'webservers']
+    options.check_mode = True
+    options.start_at_task = u'Start with くらとみ'
```

Comment:
*/me requests emoji test*

Reply:
heh.  Sure, just make sure that any emoji you put into a test aren't coming from the basic multilingual plane so that we're testing some thing different.